.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  push:
+    branches-ignore:
+      - 'release/*.*.x' # already building package when publishing
+
+jobs:
+  publish:
+    name: Build package (test)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Git repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Install Node dependencies
+        run: npm ci --no-audit --no-optional
```

Comment:
It might be interesting to cache `node_modules` between workflows, or even between branches based on a hash of the `package-lock.json`, in order to speed up CI runs.

Food for thought.

Reply:
Definetly, maybe such as https://github.com/actions/cache could help - I'm quite new to GitHub Actions though.